nrwl/nx
# feat(schematics): Add ability to ng update from Nx 1.x

This adds the new Angular CLI Migrations to Nx!

This allows a user to upgrade from a Nx 1.x workspace to a Nx 6.0 Workspace!

Ng Update Specifications: https://github.com/angular/devkit/blob/master/docs/specifications/update.md

### Update Procedure:
1. Install the latest version of `@nrwl/schematics`
```
yarn add @nrwl/schematics@latest
```
* On Local this will be 
```
yarn add ../../nx/builds/packages/schematics
```

2. Run the last Nx update
```
yarn update
```
3. Update `@angular/cli`
```
yarn add @angular/cli@latest -D
ng update @angular/cli
```

4. Update `@nrwl/schematics`
```
ng update @nrwl/schematics
```
* On Local this will be
```
ng @schematics/update:migrate --package @nrwl/schematics --collection @nrwl/schematics/migrations/migrations.json --from 1.0.0 --to 6.0.0
```

After all this, the project should be mostly functioning.

Note:
It might be ugly.

Fixes #476
